src/store/reducers/createMapReducer/tests/activeLayerIdReducer.test.js

```diff
@@ -14,7 +13,7 @@ describe('activeLayerIdReducer', () => {
   };
 
   it('should return the initial state', () => {
-    expect(activeLayerIdReducer(undefined, {})).toEqual(defaultState.MAP.activeLayerId);
+    expect(activeLayerIdReducer(undefined, {})).toEqual('');
```

Comment:
@kelvin-muchiri why not test with the `defaultState` values?

Reply:
Figured out that if someone changes the default values to a wrong value, then it will affect the integrity of the tests

Reply:
Hmm interesting, do you think the tests' integrity is effected or their outcomes? In other words: 

If someone changes the `defaultState` then do the tests a) fail to test the code properly, or b) simply fail?

If `a` then it feels like the tests need to be written in a way that their ability to test the code isn't compromised.

If `b` then I think the failing tests very vividly illustrate the impact of changing the default state (🎉) and that test snapshots should be updated alongside changes to `defaultState`. Then the question would be if we use the `defaultState` or if we decouple and maintain a `defaultState` for tests (in a fixtures file)

Reply:
It's A. The tests fail to test code properly. Also, say TDD was applied before writing the code, then it means we would not have had the `defaultState.js` file in existence, and so we would have had to write the raw value required. e.g `toEqual(' ')` or `toEqual([ ])`. What do you think?

Reply:
> It's A. The tests fail to test code properly

Ah ok, is this because of how the tests are written or the code? If it's the later then let's open a ticket to discuss the underlying issue with the code.

> say TDD was applied before writing the code, then it means we would not have had the defaultState.js file in existence

Good point, we definitely don't want the tests coupled directly to the `defaultState` for this reason, and as long as the raw values match the current values in `defaultState` then yeah it seems we still have coverage if the `defaultState` changes at all. 👍